pages/tutorials/3.0/getting-started/code-blocks.md

```diff
@@ -33,72 +42,88 @@ These paths are the same in both Debug and Release configuration, so you can set
+You might have noticed from the table that SFML modules can also depend on one another, e.g.
+sfml-graphics-s depends both on sfml-window-s and sfml-system-s.
+If you static link to an SFML library, make sure to link to the dependencies of the library in question, as well as the dependencies of the dependencies and so on.
+If anything along the dependency chain is missing, you *will* get linker errors.
 
-Additionally, because Code::Blocks makes use of GCC, the linking order *does* matter. This means that libraries that depend on other libraries have to be added to the library list *before* the libraries they depend on. If you don't follow this rule, you *will* get linker errors.
+Additionally, because Code::Blocks makes use of GCC, the linking order *does* matter.
+This means that libraries that depend on other libraries have to be added to the library list *before* the libraries they depend on.
+If you don't follow this rule, you *will* get linker errors.
 
-If you are slightly confused, don't worry, it is perfectly normal for beginners to be overwhelmed by all this information regarding static linking. If something doesn't work for you the first time around, you can simply keep trying always bearing in mind what has been said above. If you still can't get static linking to work, you can check the [FAQ](https://www.sfml-dev.org/faq.php#build-link-static "Go to the FAQ page") and the [forum](http://en.sfml-dev.org/forums/index.php?board=4.0 "Go to the general help forum") for threads about static linking.
+If you are slightly confused, don't worry, it is perfectly normal for beginners to be overwhelmed by all this information regarding static linking.
+If something doesn't work for you the first time around, you can simply keep trying always bearing in mind what has been said above.
+If you still can't get static linking to work, you can check the [FAQ](https://www.sfml-dev.org/faq.php#build-link-static "Go to the FAQ page") and the [forum](http://en.sfml-dev.org/forums/index.php?board=4.0 "Go to the general help forum") for threads about static linking.
 
-If you don't know the differences between dynamic (also called shared) and static libraries, and don't know which one to use, you can search for more information on the internet. There are many good articles/blogs/posts about them.
+If you don't know the differences between dynamic (also called shared) and static libraries, and don't know which one to use, you can search for more information on the internet.
+There are many good articles/blogs/posts about them.
 
-Your project is ready, let's write some code now to make sure that it works. Add a "main.cpp" file to your project, with the following code inside:
+Your project is ready, let's write some code now to make sure that it works.
+Add a "main.cpp" file to your project, with the following code inside:
```

Comment:
```suggestion
Add a `main.cpp` file to your project, with the following code inside:
```

Reply:
I went through other instances of `main.cpp` and made sure they're monospaced to maintain consistency.